es/news/_posts/2023-03-30-ruby-3-1-4-released.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: news_post
+title: "Publicado Ruby 3.1.4 Released"
+author: "nagachika"
+translator: vtamara
+date: 2023-03-30 12:00:00 +0000
+lang: es
+---
+
+Se ha publicado Ruby 3.1.4.
+
+Ver detalles en la [publicación en GitHub](https://github.com/ruby/ruby/releases/tag/v3_1_4).
```

Comment:
```suggestion
Se ha publicado Ruby 3.1.4.

Esta versión incluye correcciones de seguridad.
Por favor vea detalles en los siguientes temas.

* [CVE-2023-28755: Vulnerabilidad ReDoS en URI]({%link es/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md %})
* [CVE-2023-28756: Vulnerabilidad ReDoS en Time]({%link es/news/_posts/2023-03-30-redos-in-time-cve-2023-28756.md %})

Ver más detalles en la [publicación en GitHub](https://github.com/ruby/ruby/releases/tag/v3_1_4).
```

Reply:
The original post in english doesn't include that information, and according to the posts of the vulnerabilites Ruby 3.1.4 is not vulnerable (it is required just to update the gems uri and time).

Reply:
Hey @vtamara, from what I see, https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2023-03-30-ruby-3-1-4-released.md includes that information.